### ISSUE TITLE
[fastify-swagger] is it possible to generate swagger-schema static pages based on route descriptions object { route: schema}?

### ISSUE THREAD

**Issue by budarin** [fastify/help#740]
## 💬  Question here 

We use json-rpc protocol so we use one universal route to handle all api requests to db
we use routes description object {route: schema} and use custom ajv instance to validate parameters, pass it to one db function and then send the result to clients

so we don't have real routes - so we can not use fastify-swagger plugin, bun we have all information the plugin is need

is it possible to generate static swagger schema description pages based on the object providing the path and schema?

## Your Environment

- *node version*: v18.6.0
- *fastify version*: 4.4.0
- *os*: any

**Comment by Uzlopak** [comment]
I dont think that this is easily possible. Probably you have to fork swagger-ui or something like rapidocs and modify it to handle each of json-rpc cases.
